tests/e2e/utils/plugin-utils.js

```diff
@@ -107,43 +84,3 @@ export const deleteZip = async ( zipFilePath ) => {
 	} );
 	console.log( `\u2714 Successfully deleted!` );
 };
-
-/**
- * Get the download URL of the release zip from GitHub
- *
- * @param string version The version to be tested.
- * @return string Download URL for the release zip file.
- */
-export const getReleaseZipUrl = async ( version ) => {
```

Comment:
This function is still being used in [`global-teardown.js`](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/6ff08d33b76ab21a23c88248eb259cbce0efbe99/tests/e2e/config/global-teardown.js#L4).

Reply:
I've refactored the clean up of downloaded files [here](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2522/commits/9963871c5226e070a1e4f306f39dbb8075bc1b79). Now it'll delete the entire `tmp` folder after the tests instead of individual files.